include/unifex/stream_concepts.hpp

```diff
@@ -19,15 +19,8 @@
 #include <unifex/sender_concepts.hpp>
 
 namespace unifex {
-namespace cpo {
 
 inline constexpr struct next_cpo {
-  template <typename Stream>
-  friend auto tag_invoke(next_cpo, Stream& s) noexcept(noexcept(s.next()))
-      -> decltype(s.next()) {
-    return s.next();
-  }
-
```

Comment:
what is the reason for removing the member function impls for next and cleanup?

Reply:
There were two reasons for this:
1. The `tag_invoke` paper talked about the potential for conflict when using ADL names or method names are used as customisation points and so I was trying to minimise the number of member function names that CPOs dispatch to.
2. I initially encountered naming conflicts when converting adapter code to call the CPO without the `cpo::` namespace qualification (the member name now shadows the unqualified CPO name) and so I started changing some of the member functions to tag_invoke() overloads to avoid this naming conflict. However, this could also have been addressed by prefixing the call with `unifex::` to disambiguate (I ended up doing this in some of the other CPOs in other commits).

Reply:
Ok, I've updated the CPOs to restore the behaviour of `next()` and `cleanup()` dispatching to member functions.